concourse/scripts/pipelines-cloudfoundry.sh

```diff
@@ -67,6 +67,25 @@ elif [ "${DEPLOY_ENV}" = "stg-lon" ]; then
   deploy_env_tag_prefix="v[0-9]*" # this matches all tags created by release ci
 fi
 
+case $DEPLOY_ENV in
+prod-lon)
+  # Flag of Greater London
+  BACKGROUND_IMAGE_URL="https://upload.wikimedia.org/wikipedia/commons/2/25/Flag_of_Greater_London.svg"
```

Comment:
I like that you've gone and searched out the real flags, but I honestly would have no idea which one I was looking at from that :( Could we use 🏴󠁧󠁢󠁥󠁮󠁧󠁿  and 🇮🇪  ?

Reply:
I've replaced that with the City of London flag: 

![Flag of the City of London](https://upload.wikimedia.org/wikipedia/commons/thumb/c/c7/Flag_of_the_City_of_London.svg/2880px-Flag_of_the_City_of_London.svg.png)

Reply:
Could we not use https://secretldn.com/wp-content/uploads/2017/02/london-classic-iconic-red-phoneboxes.jpg for London and https://www.irishcentral.com/uploads/article/20853/cropped_st_patricks_day_shamrocks___getty.jpg?t=1583226320 for Dublin?

I dislike flags

Reply:
Some fantastic bike shedding potential here ... _follows and gets popcorn_ :popcorn: :smile:

Reply:
Went with Lee's ideas, they're good